client/retry/backoff_test.go

```diff
@@ -119,13 +119,28 @@ func TestBackofferWithLog(t *testing.T) {
 	lg := newZapTestLogger(conf)
 	log.ReplaceGlobals(lg.Logger, nil)
 
-	bo := InitialBackoffer(time.Millisecond*10, time.Millisecond*100, time.Millisecond*1000, WithLogTimes(10))
+	bo := InitialBackoffer(time.Millisecond*10, time.Millisecond*100, time.Millisecond*1000, withMinLogInterval(time.Millisecond*100))
 	err := bo.Exec(ctx, testFn)
 	re.ErrorIs(err, errTest)
 
-	m1 := lg.Message()
-	rfc := `["call PD API failed and retrying"] [api=testFn] [retry-time=10] [error=test]`
-	re.Contains(m1, rfc)
+	ms := lg.Messages()
+	len1 := len(ms)
+	re.Len(ms, 9)
```

Comment:
Can you add more information to explain why care 5 and 13?(such as (1+2+4+8)%10=5)